src/einsteinpy/symbolic/predefined/vacuum_solutions.py

```diff
@@ -55,13 +55,11 @@ def Kerr(c=constants.c, sch=symbols("r_s"), a=symbols("a")):
         1 - (sch * r / Sigma),
         -Sigma / (Delta * c2),
         -Sigma / c2,
-        -(
-            (r ** 2 + a ** 2 + (sch * r * (a ** 2) * (sin(theta) ** 2) / Sigma))
-            * (sin(theta) ** 2)
-        )
-        / c2,
+        (Delta * a ** 2 * sin(theta) ** 2 - (r ** 2 + a ** 2) ** 2)
+        * sin(theta) ** 2
+        / (Sigma * c2),
     ).tolist()
-    metric[0][3] = metric[3][0] = sch * r * a * (sin(theta) ** 2) / (Sigma * c)
+    metric[0][3] = metric[3][0] = 2 * sch * r * a * (sin(theta) ** 2) / (Sigma * c)
```

Comment:
```suggestion
    metric[0][3] = metric[3][0] = sch * r * a * (sin(theta) ** 2) / (Sigma * c)
```

2 in the expression is coming from both `d(phi)dt` and `dtd(phi)`. So in a tensorial form, we have to redistribute the 2 again.

Reply:
Also, metric tensor in classical GR is symmetric. In fact most of the tensors in physics is symmetric